### PR TITLE
2755: Build /openstack/install page

### LINKED ISSUE #2755: Build /openstack/install page
* build in beta branch
* move /download/cloud to /openstack/install
* update with [copy doc](https://docs.google.com/document/d/10vJnhP8zjXOdtgLUnTrfKOWAHJflxdvRgAgjM5DbAZ8/edit#)

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Moved /download/cloud/index.html to /openstack/install/index.html
- Updated copy according to [copydoc](https://docs.google.com/document/d/10vJnhP8zjXOdtgLUnTrfKOWAHJflxdvRgAgjM5DbAZ8/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/install
- Check that it matches the [copydoc](https://docs.google.com/document/d/10vJnhP8zjXOdtgLUnTrfKOWAHJflxdvRgAgjM5DbAZ8/edit#)

## Issue / Card

Fixes #2755

### CODE REVIEW

File: templates/openstack/install/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          <p>Already a Landscape Dedicated Server customer? Upgrading is simple, see the instructions in the <a class="p-link--external" href="https://help.landscape.canonical.com/LDS/ReleaseNotes">release notes</a>.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+{% include "download/server/_other-ways.html" %}
+
+{% include "shared/contextual_footers/_contextual_footer.html" with first_item="_cloud_bootstack" second_item="_download_cloud_bootstack" third_item="_download_documentation" %}
```
Comment (pmahnke) [inline templates/openstack/install/index.html]: wrong middle contextual_footer

should be 

**Growing your cloud?**
Ubuntu Advantage enterprise support agreements cover OpenStack and include the Landscape licenses needed for the Autopilot in larger cloud deployments.
[ Buy now ]
Or contact our sales team ›

File: navigation.yaml [hunk 2]
```diff
@@ -39,6 +39,15 @@ openstack:
   children:
     - title: Overview
       path: /openstack
+    - title: Install
+      path: /openstack/install
+
+      children:
```
Comment (pmahnke) [inline navigation.yaml]: these are backwards... and not appearing

**Comment by spencerbygraves** [comment]
Should this div be below the secondary nav? The height of it looks wrong at the moment. Thanks.

image

**Comment by kwm14** [comment]
@Caleb-Ellis couple more design changes 😄 

### Hero
- [ ] add `is-light` to p-strip so the background inherits grey background

### Single machine / Cluster
- [ ] remove `is-light` to p-strip so the background is default white

### Visual
screen shot 2018-03-01 at 14 38 14

File: navigation.yaml (hunk 2 above)
Comment (Caleb-Ellis) [reply navigation.yaml]: I've fixed the backwardness, but not them not appearing. This is a bug with the current state of the secondary nav (basically I forgot tertiary breadcrumbs are a thing) but they'll be fixed in #2771

**Comment by Caleb-Ellis** [comment]
@pmahnke I've used the correct contextual footer, but the one from the /openstack/index.html [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#) instead because it doesn't mention Autopilot.

@spencerbygraves Yeah this a bug with the secondary nav that will be fixed once #2771 is merged (compare [here](http://www.ubuntu.com-pr-2772.run.demo.haus/download/server) and [here](http://www.ubuntu.com-pr-2771.run.demo.haus/download/server))

@kwm14 fixed

**Comment by kwm14** [comment]
Thanks @Caleb-Ellis looking good. As discussed secondary nav background is meant to be white, but that is on another branch so once merged it will be fixed 👍 

Looking at _Single machine/Cluster_ section today I think it would look better as separate cards to make it more clear we're offering 2 solutions.

### Single machine / Cluster
- [ ] separate solutions into two cards using **Basic card** styling

### Visual
screen shot 2018-03-02 at 10 31 26

**Comment by kwm14** [comment]
Updates look good. Design 👍

**Pull request merged**